### PR TITLE
339 electric power facility repair cost analysis

### PR DESCRIPTION
**Pull request by longshuicy**
#### How to test: 
- run test **tests/pyincore/analyses/epfrepaircost/test_epfrepaircost.py** as it is

~~[data.zip](https://github.com/IN-CORE/pyincore/files/11491951/data.zip)~~

~~- Once finalized I can upload those datasets to incore services~~

#### Sample output
[seaside_epf_repair_cost.csv](https://github.com/IN-CORE/pyincore/files/11491953/seaside_epf_repair_cost.csv)

### CODE REVIEW

**Review by ywkim312 — APPROVED** [review]
test ran successfully with the same output

**Review by ylyangtw — APPROVED** [review]
Tested it and got the same output. Approved.

File: pyincore/analyses/epfrepaircost/epfrepaircost.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+            rc["p_budget"] = ','.join(repair_cost)
+            rc["q"] = ','.join(repair_cost)
+
+            repair_costs.append(rc)
+
+        return repair_costs
+
+    def get_spec(self):
+        """Get specifications of the epf damage analysis.
```
Comment (longshuicy) [inline pyincore/analyses/epfrepaircost/epfrepaircost.py:153]: uh oh typo...
note to myself change to epf repair cost

File: pyincore/analyses/epfrepaircost/epfrepaircost.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2019 University of Illinois and others. All rights reserved.
```
Comment (navarroc) [inline pyincore/analyses/epfrepaircost/epfrepaircost.py:1]: Can you put the current year?

File: pyincore/analyses/epfrepaircost/epfrepaircost.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+import concurrent.futures
+from itertools import repeat
+
+from pyincore import AnalysisUtil, GeoUtil
+from pyincore import BaseAnalysis, HazardService, FragilityService
+from pyincore.models.dfr3curve import DFR3Curve
```
Comment (navarroc) [inline pyincore/analyses/epfrepaircost/epfrepaircost.py:12]: Looks like there are some unused imports, can you remove those?

File: pyincore/analyses/epfrepaircost/epfrepaircost.py [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+                output.extend(ret1)
+
+        return output
+
+    def epf_repair_cost_bulk_input(self, epfs):
+        """Run analysis for multiple epfs.
+
+        Args:
+            epfs (list): Multiple epfs from input inventory set.
+            hazard_type (str): A type of hazard exposure (earthquake, tsunami, tornado, or hurricane).
```
Comment (navarroc) [inline pyincore/analyses/epfrepaircost/epfrepaircost.py:83]: Looks like this can be removed

**Review by navarroc — APPROVED** [review]

**Pull request merged**